applications/nsmrs/go.mod

```diff
@@ -14,6 +14,8 @@ require (
 )
 
 replace (
+	github.com/networkservicemesh/networkservicemesh => ../../
+	github.com/networkservicemesh/networkservicemesh/applications/nsmrs => ./
```

Comment:
Please let's undo these lines and let's solve this as a separate problem because it doesn't solve the problem with  diff after run unit tests.
Please take a look at https://github.com/networkservicemesh/networkservicemesh/pull/2097
Also, these lines will break artifacts saving for helm tests:
https://120638-127937836-gh.circle-artifacts.com/0/home/circleci/project/.tests/cloud_test/aws-5/004-Example-helm-icmp%20kernel-forwarder-run.log
NOTE:
```
go: github.com/networkservicemesh/networkservicemesh/test@v0.0.0-00010101000000-000000000000 requires
	github.com/networkservicemesh/networkservicemesh/applications/nsmrs@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000
.mk/k8s.mk:151: recipe for target 'k8s-save-artifacts-only-master' failed
make: *** [k8s-save-artifacts-only-master] Error 1
```